hamster_gtk/helpers.py

```diff
@@ -135,3 +138,39 @@ def get_config_instance(fallback_config_instance, app_name, file_name):
 def get_resource_path(file_name):
     """Return path to a resource file."""
     return os.path.join(os.path.dirname(__file__), 'resources', file_name)
+
+
+def serialize_activity(activity):
+    """Provide a serialised activity string representation."""
```

Comment:
In the comments you use ‘serialised’ while the rest of the app uses American spelling. This should probably be unified.